### DISCUSSION TITLE
WSL does not have supervisor

### DISCUSSION THREAD

**Discussion by Wulfheart** [anystack-sh/porter#18] [Q&A]
Hey,

this library looks great. However, my WSL2 does not have supervisor installed and I couldn't make it running. Is there any other way to run this project?

**Comment by PhiloNL** [comment]
Hey @Wulfheart,

Thanks! I have yet to have the chance to test this on other operating systems. I'm using macOS primarily, and I don't expect to be able to test this on different systems soon, unfortunately. Porter is a layer on top of Supervisor, so this is a requirement. If you find a solution, I would love to know more.

**Comment by HassanZahirnia** [comment]
Hey @Wulfheart 
I'm also using WSL2 for dev, and I've had no problem installing supervisor. Why couldn't make it running?
You can just install supervisor like you'd normally do in any Ubuntu machine. Except every time you start WSL2, you have to manually start the supervisor service:
```
sudo service supervisor start
```

**Reply by Wulfheart** [reply]
Somehow I managed to remove it. At least that's what I think. :D

```
supervisor: unrecognized service
```

**Reply by Wulfheart** [reply]
Ah. I had to add the universe repository...

**Reply by HassanZahirnia** [reply]
Glad you figured it out 😊👍

**Reply by Wulfheart** [reply]
Yeah. Somehow hashicorp adds it own repositories and overwrites everything else. Quite bold of them tbh.
